beamer/tests/agent/test_request.py

```diff
@@ -363,20 +363,6 @@ def test_agent_only_claim_once_after_restart(request_manager, token, agent, dire
                     break
 
 
-def test_event_fetcher_progress_without_events(agent, direction):
-    with Sleeper(5) as sleeper:
-        while ape.chain.chain_id not in agent.get_context(direction).latest_blocks:
-            sleeper.sleep(0.1)
-    for _ in range(5):
-        ape.chain.mine(1)
-        with Sleeper(5) as sleeper:
-            while (
-                agent.get_context(direction).latest_blocks[ape.chain.chain_id].number
-                != ape.chain.blocks[-1].number
```

Comment:
What does `[-1]` mean in `ape.chain.blocks[-1].number`

Reply:
latest block number in this case

Reply:
i mean this is `ape.chain.blocks[-1]` latest block

Reply:
Ah ok so -1 does not mean `latest - 1`?

Reply:
no directly the latest like in python list

Reply:
got it, just wanted to be crystal clear. thanks